docusaurus.config.js

```diff
@@ -41,10 +41,11 @@ const config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       ({
         docs: {
-          sidebarPath: './sidebars.js',
-          // Please change this to your repo.
-          editUrl:
-          'https://github.com/open-sauced/intro',
+          sidebarPath: require.resolve('./sidebars.js'),
```

Comment:
**question:** Is there any reason to modify the `sidebarPath`? I don't see it's related to the issue.